lib/pure/ioselects/ioselectors_epoll.nim

```diff
@@ -81,7 +81,7 @@ proc newSelector*[T](): Selector[T] =
   # Start with a reasonable size, checkFd() will grow this on demand
   const numFD = 1024
 
-  var epollFD = epoll_create(MAX_EPOLL_EVENTS)
+  var epollFD = epoll_create1(O_CLOEXEC)
```

Comment:
epoll_create1 was added to the Linux in version 2.6.27. Is it ok for Nim's minimal supported Linux version?
http://man7.org/linux/man-pages/man2/epoll_create.2.html

Reply:
It's released over a decade ago, I don't see why not :)

Reply:
Hmm, but isn't Linux 2.4 still widely used for embedded development?

Reply:
We don't have to care about that, since ioselectors_epoll uses eventfd(2) which is added in Linux 2.6.22.